### ISSUE TITLE
Generic container for non-docker local boefjes

### ISSUE THREAD

**Issue by Donnype** [minvws/nl-kat-coordination#3698]
Blocked by: #3861 
Subticket of https://github.com/minvws/nl-kat-coordination/issues/3593.

---

TODO
- [x] Create a generic image and adapter that handles the `oci_arguments` being the path to the targeted module
- [x] Rewrite all non-docker local boefjes by making the `run()` method accept a dict and hence drop the `BoefjeMeta` type hint as well. Also we need to add the `requirements.txt` back in the boefje directory.
- [x] Drop the requirements that are not needed anymore from the boefjes `pyproject.toml`
- [x] ~Create migration for boefjes in the `boefje` table: changing the boefjes.json for static boefjes means the db is out of sync for entries for that boefje. We need to either keep these aligned or make sure we ignore the fields for static database entries.~ **-> we should ignore the database entries for the time being**

**Comment by Donnype** [comment]
Closing in favor of https://github.com/minvws/nl-kat-coordination/issues/3861 since I have this code merged into that branch and we need to merge them together.
